easybuild/easyconfigs/a/ABySS/ABySS-2.0.3-intel-2017b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABySS'
+version = '2.0.3'
+
+homepage = 'http://www.bcgsc.ca/platform/bioinfo/software/abyss'
+description = """Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler"""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/bcgsc/abyss/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['abyss-2.0.3-BranchGroup.patch']
```

Comment:
@murfreesboro Some nitpicking here: please use `ABySS` to keep it in sync with the `name` in the easyconfig (which requires renaming the patch file), and use `%(version)s` rather than hardcoding `2.0.3` when it matches the `version` used in the easyconfig file:

```python
patches = ['ABySS-%(version)s-BranchGroup.patch']

Reply:
changed the patch file name and modified the eb file.